src/OpenTelemetry.Instrumentation.Process/README.md

```diff
@@ -40,8 +40,53 @@ using var meterProvider = Sdk.CreateMeterProviderBuilder()
     .Build();
 ```
 
+Refer to [Program.cs](../../examples/runtime-instrumentation/Program.cs) for a
+complete demo.
+
 ## Metrics
 
+### process.memory.usage
+
+The amount of physical memory allocated for this process.
+
+| Units | Instrument Type   | Value Type |
+|-------|-------------------|------------|
+|  `By` |  ObservableGauge  | `Double`   |
+
+### process.memory.virtual
+
+The amount of virtual memory allocated for this process
+that cannot be shared with other processes.
+
+| Units | Instrument Type   | Value Type |
+|-------|-------------------|------------|
+|  `By` |  ObservableGauge  | `Double`   |
+
+### process.cpu.time
+
+Total CPU seconds broken down by states.
+
+| Units | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|-------|-------------------|------------|------------------|------------------|
+|  `s`  | ObservableCounter | `Double`   | state            | user, system     |
+
+### process.cpu.utilization
+
+Difference in process.cpu.time since the last measurement,
+divided by the elapsed time and number of CPUs available to the process.
+
+| Units | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|-------|-------------------|------------|------------------|------------------|
+|  `1`  | ObservableCounter | `Double`   | state            | user, system     |
+
+### process.threads
+
+Process threads count.
+
+| Units           | Instrument Type   | Value Type |
+|-----------------|-------------------|------------|
+| `{threads}`     | ObservableCounter | `Int32`    |
```

Comment:
suspicious on using Counter here.... the count of threads always go up/stay same only?

Reply:
i think the description must be more detailed explaining what does this number represent.

Reply:
Copy-pasta error; the source code is correct. Fixed.

Do you think by adding the API used will solve the lack of detail issue for the description of this instrument?

Reply:
no...the description should be very clear.